Checkbox+Button (#356)
reflex-dev/reflex-web: docs/library/forms/checkbox.md

```python exec
import reflex as rx
import reflex.components.radix.themes as rdxt
from pcweb.templates.docpage import style_grid, docdemo_from
```

# Checkbox

Checkboxes allow users to select one or more items from a set. This component uses Radix's [checkbox](https://radix-ui.com/primitives/docs/components/checkbox) component.

## Basic Example

```python demo
rdxt.text(
  rdxt.flex(
    rdxt.checkbox(default_checked=True),
    "Agree to Terms and Conditions", 
    gap="2",
  ),
  as_="label",
  size="2",
)
```

## Props

### Disabled

The `disabled` prop disables the button, by default it is `False`.

```python demo
rx.hstack(
    rdxt.checkbox(),
    rdxt.checkbox(disabled=True),
)
```

## Triggers

### OnCheckedChange

> The `on_change` trigger is called when the checkbox is clicked.

Reviewer: ```suggestion
The `on_checked_change` trigger is called when the checkbox is clicked.
```

```python demo
rdxt.checkbox(default_checked=True, on_checked_change=rx.window_alert("Checked!"))
```

## Styling

> ### Color
> 
> The `color` prop changes the color of the checkbox. The default color is `primary`.

Reviewer: ```suggestion
### Color Scheme

The `color_scheme` prop changes the color of the checkbox. The default color maps to the `accent_color` in the app's `theme`.
```

```python demo
rx.hstack(
    rdxt.checkbox(color_scheme="grass", default_checked=True),
    rdxt.checkbox(color_scheme="indigo", default_checked=True),
    rdxt.checkbox(color_scheme="orange", default_checked=True),
    rdxt.checkbox(color_scheme="tomato", default_checked=True),
)
```

### High-constrast

```python demo
rx.vstack(
    rx.hstack(
      rdxt.checkbox(color_scheme="grass", default_checked=True),
      rdxt.checkbox(color_scheme="indigo", default_checked=True),
      rdxt.checkbox(color_scheme="orange", default_checked=True),
      rdxt.checkbox(color_scheme="tomato", default_checked=True),
  ),
  rx.hstack(
      rdxt.checkbox(color_scheme="grass", default_checked=True, high_contrast=True),
      rdxt.checkbox(color_scheme="indigo", default_checked=True, high_contrast=True),
      rdxt.checkbox(color_scheme="orange", default_checked=True, high_contrast=True),
      rdxt.checkbox(color_scheme="tomato", default_checked=True, high_contrast=True),
  )
)
```

### Size

> The `size` prop changes the size of the button. The default size is `2`.

Reviewer: ```suggestion
The `size` prop changes the size of the checkbox. The default size is `2`.
```
